mysql.env

```diff
@@ -0,0 +1,5 @@
+MYSQL_ROOT_PASSWORD=password
+MYSQL_DATABASE=project
+MYSQL_USER=admin
+MYSQL_PASSWORD=password
+MYSQL_PORT=3306
```

Comment:
I assume there is no routing for overwriting this for production yet, right? We should probably add that as a user story to a future sprint.

Reply:
We could create another `docker-compose.yml` file. For example one for production named as `docker-compose.production.yml` and there we could overwrite the env_file. But yes, let's create an user story for this one.

Reply:
👍